src/components/common/TextBlock.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+import '@components/VerticalBox.vue'
+</script>
+
+<template>
+  <div class="center-wrapper">
+    <VerticalBox>
+      <h3 class="left-wrapper" style="padding-top: 1rem; padding-inline: 1rem; text-align: left">
+        {{ title }}
+      </h3>
+      <p>
+        <slot></slot>
+      </p>
+    </VerticalBox>
+  </div>
+</template>
+
+<script lang="ts">
```

Comment:
You should try using the composition API sometime! It's very freeing, and gives you access to the very powerful composables part of the API.

Reply:
Refactored TextBlock to use it, will make a conscious effort to use it more!